docs/configuration/configuration.md

```diff
@@ -76,7 +76,8 @@ global:
   [ rule_query_offset: <duration> | default = 0s ]
 
   # The labels to add to any time series or alerts when communicating with
-  # external systems (federation, remote storage, Alertmanager).
+  # external systems (federation, remote storage, Alertmanager). 
+  # Note that external_labels are always enabled in release-3.0.
```

Comment:
External labels are always added. what we need here is a short description of how env variables are expanded. see `docs/feature_flags.md`.

Reply:
How about add a example just like `# Example: exported_instance:127.0.0.1`?

Reply:
No this is not sufficient. External labels have always been enabled. This change here is about Prometheus expanding environment variable references in the external labels config. This feature used to be behind a feature flag, now this expanding behavior is enabled by default.
So we need to document how this environment variable expansion works. Here we need to add something like
```
Environment variable references `${var}` or `$var` are replaced according to the values of the current environment variables. References to undefined variables are replaced by the empty string.
The `$` character can be escaped by using `$$`.
```